File: HW2/Codes/Oz_Elifsena.dart
void main() {
  
  // This aims to print numbers from 0 to 4 inclusive
  print("\nConditional");
  for (int i = 0; i <=4; i++) {
    print("i: $i");
  }
 
  // This aims to print numbers from 0 to 4 inclusive
  print("\nUnconditional");
  for (int i = 0; i <= 4; i++) {
    print("i: $i");
    
    if (i == 3) {
      print("Exit loop");
      break;  // When i is 3 the loops is exited
    }
  }

  print("\nUnlabeled");
  for (int i = 0; i < 3; i++) { // This aims to print numbers from 0 to 2 inclusive
    print("i: $i");
    for (int j = 3; j <= 4; j++) { // This aims to print numbers from 3 to 4 inclusive
      if (j == 4) {
        break; // j is 4  // When j is 4 the inner loop is exited
      }
      print("j: $j"); 
    }
  }
  
  print("\nLabeled");
  labeledLoop: for (int i = 0; i < 3; i++) { // This aims to print numbers from 0 to 2 inclusive
    print("i: $i");
    for (int j = 3; j <= 4; j++) { // This aims to print numbers from 3 to 4 inclusive
      if (j == 4) {
        break labeledLoop; // j is 4 // When j is 4 the outer loop (labeledLoop) is exited
    }
    print("j: $j"); 
    }
  }  
}

File: HW2/Codes/Oz_Elifsena.rs
fn main() {
    println!("\nConditional");
    for i in 0..5 { // This aims to print numbers from 0 to 4 inclusive
        println!("i: { }", i);
    }
    
    println!("\nUnconditional");
    for i in 0..5 {  // This aims to print numbers from 0 to 4 inclusive
        println!("i: { }", i);
        if i == 3 {
            println!("Exit Loop");
            break  // When i is 3 the loops is exited
        }
    }
    
    println!("\nUnlabeled");
    for i in 0..3 { // This aims to print numbers from 0 to 2 inclusive
        println!("i: { }", i);
        for j in 3..5 {  // This aims to print numbers from 3 to 4 inclusive
            if j == 4 {
                break; // When j is 4 the inner loop is exited
            }
            println!("j: { }", j);
        }
    }
    
    println!("\nLabeled");
    'labeledLoop: for i in 0..3 { // This aims to print numbers from 0 to 2 inclusive
        print("i: $i");
        println!("i: { }", i);
        for j in 3..5 { // This aims to print numbers from 3 to 4 inclusive
            if j == 4 {
                break 'labeledLoop; // When j is 4 the outer loop (labeledLoop) is exited
            }
            println!("j: { }", j);
        }
    }
}